### DISCUSSION TITLE
G-Helper Ambient Mode

### DISCUSSION THREAD

**Discussion by muhannadafana** [seerge/g-helper#1731] [Q&A] [answered]
I have my keyboard lighting set to ambient mode, and it works fine except that  it seems it only changes the color of the keyboard to the color of a small point on the left middle of the screen. Is there any way to change that? As in, change to the average color of the screen or set a different point?

**Comment by seerge** [comment]
@muhannadafana hello, it takes average color of your screen by making a screenshot :) so it's not 1 dot.

Run some bright youtube video on fullscreen with changing colors to test ;)

**Reply by muhannadafana** [reply]
mine seems to only do the left side

https://github.com/seerge/g-helper/assets/153657909/8d6772fe-9581-4082-b4e5-3eba5b4eb9bd

**Reply by seerge** [reply]
@muhannadafana i see that you have animated wallpaper, is it some exta app ? if so it could be actually replacing real screen and just confusing screenshots. 

Anyway I can't do much here , as screenshot method in windows is super standard, if by some (software) reasons it doesn't return correct screenshot in your case, that's how things are.

**Reply by muhannadafana** [reply]
it does is whether or not the wallpaper software is on or not (wallpaper engine), ill just try and figure it out then but its not a big deal.

**Reply by muhannadafana** [reply]
i just updated to 0.137 and it works fine now

**Reply by seerge** [reply, marked as answer]
@muhannadafana cool, you can also test a build from this topic https://github.com/seerge/g-helper/discussions/1736 i have adjusted a way an average color is calculated a bit, it may help

**Reply by muhannadafana** [reply]
wow that works much better actually, thank you!
